src/overlays/render_settings.rs

```diff
@@ -343,6 +350,39 @@ impl Overlay for RenderSettingsOverlay {
 
                 maps.current_map = current_map;
 
+                // Move the Globals from the old scene to the new scene
+                if map_changed {
+                    // We have learned the power to Take worlds
+                    let mut old_scene = take(&mut maps.map_mut(old_map_index).unwrap().scene);
+
+                    let mut ent_list = vec![];
+
+                    for (entity, global) in old_scene.query::<Option<&Global>>().iter() {
+                        if let Some(g) = global {
+                            if g.0 {
+                                ent_list.push(entity);
+                            }
+                        }
+                    }
```

Comment:
Querying a component as optional and then checking it is redundant
```suggestion
                    for (entity, global) in old_scene.query::<&Global>().iter() {
                        if g.0 {
                            ent_list.push(entity);
                        }
                    }
```